libbpf-rs/src/query.rs

```diff
@@ -98,101 +98,305 @@ fn name_arr_to_string(a: &[c_char], default: &str) -> String {
     }
 }
 
+#[derive(Debug, Clone, Default)]
+#[repr(C)]
+struct LineInfoRaw {
+    insn_off: u32,
+    file_name_off: u32,
+    line_off: u32,
+    line_col: u32,
+}
+
+#[derive(Debug)]
+pub struct LineInfo {
```

Comment:
Why do we need this type and can't use https://docs.rs/libbpf-sys/1.2.1+v1.2.0/libbpf_sys/struct.bpf_line_info.html? Isn't that what it is?

Reply:
Sorry, this comment was meant to apply to `LineInfoRaw`.